upp/stages/resampling.py

```diff
@@ -54,9 +114,10 @@ def __init__(self, config):
 
     def countup_select_func(self, jets, component):
         num_jets = int(len(jets) * component.sampling_fraction)
```

Comment:
Should we add a check here that `upscale_pdf` is equal to 1 and raise and error if not?

Reply:
Done